.github/workflows/pkg.yaml

```diff
@@ -59,8 +59,8 @@ jobs:
       with:
         tag_name: ${{ steps.get_source_tag.outputs.SOURCE_TAG }}
         release_name: Release ${{ steps.get_source_tag.outputs.SOURCE_TAG }}
-        draft: false
-        prerelease: true
+        draft: true
+        prerelease: false
```

Comment:
Instead of using a `prerelease`, I changed it to `draft`. The only reason is that I found an action to publish `draft` releases, but I couldn't find one to publish `prereleases`.

I think _drafts_ are ok for our use case, but please let me know if you have any concerns.